Multisig/Data/Services/SafeTransactionService.swift

```diff
@@ -55,8 +55,16 @@ class SafeTransactionService {
         return try httpClient.execute(request: SafeBalancesRequest(address: address))
     }
 
-    func transactions(address: Address, offset: Int = 0, limit: Int = 100) throws -> TransactionsRequest.Response {
-        return try httpClient.execute(request: TransactionsRequest(address: address, limit: limit, offset: offset))
+    func transactions(address: Address?, offset: Int = 0, limit: Int = 20) throws -> TransactionsRequest.Response {
+        return try httpClient.execute(request: TransactionsRequest(address: address!, limit: limit, offset: offset))
+    }
+
+    func loadTransactionsPage(url: String) throws -> TransactionsRequest.Response {
```

Comment:
Just curious, is there a reason for using the url instead of the `nextPageRequest` from the previous response?

Reply:
Yes, I didn't want to have reference for the request (which is part of logic layer) in view model, I preferred to have reference for the url as strings,

Reply:
... then probably those "nextRequest" and "previousRequest" - are they needed at all?

Reply:
No not used, but I thought to keep them for now in case we changed something later